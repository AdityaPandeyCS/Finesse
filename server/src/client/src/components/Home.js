import React from 'react';

const Home = () => {
    return (
        <div>
            <p>Site used for administrative uses only.</p>
        </div>
    );
};

export default Home;
