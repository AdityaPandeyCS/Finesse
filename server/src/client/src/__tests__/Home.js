import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe("Home page", () => {
    it('It should load home admin page', () => {
        const component = renderer.create(
            <Home/>
        );
        let tree = component.toJSON();
        console.log(tree);
        expect(tree).toMatchSnapshot();
    });
});

