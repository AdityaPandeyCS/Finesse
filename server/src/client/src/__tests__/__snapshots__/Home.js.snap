// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page It should load home admin page 1`] = `
<div>
  <p>
    Site used for administrative uses only.
  </p>
</div>
`;
